release_package.yml

```diff
@@ -10,8 +10,30 @@
   any_errors_fatal: false # don't bomb out the entire playbook if one host (i.e. package) fails
   gather_facts: no
   tasks:
-    - include: tasks/setup_sources.yml
+    - name: 'Get latest package info from copr'
+      command: "copr-cli get-package @theforeman/tfm-ror51 --name {{ inventory_hostname }} --with-latest-build"
```

Comment:
What if there is no such puckage in the copr repo yet?

Reply:
Touche sir.